src/util/preferredLanguage.js

```diff
@@ -0,0 +1,15 @@
+import { hasExtension, getExtension } from "./extensions";
+
+export const getPreferredLanguage = hasExtension("getPreferredLanguage") ?
+  getExtension("getPreferredLanguage")
```

Comment:
Extensions are currently stored as a JSON file and so can't define functions. (One day we will be able to provide functions, but this is not currently the case.)